js/retry-timeout-playground.js

```diff
@@ -46,6 +47,24 @@ var retryTimeoutPlayground = function() {
             $(retryTickContainer).empty();
         },
 
+        setMaxDurationOnTimeline: function(maxDurationValueInMS) {
+            // Convert the inputted MS value to Seconds
+            var maxDurationSeconds = Math.round((maxDurationValueInMS/1000) * 10)/10;
+            if (maxDurationSeconds === 0) {
+                // If the converted value is less than .1s, convert to the 
+                // smallest amount of time greater than 0 seconds.
+                var convertedToSeconds = maxDurationValueInMS/1000;
+                var decimalPoints = 100;
+                while (maxDurationSeconds === 0) {
```

Comment:
if `maxDurationValueInMS` is 0, won't this loop infinitely?

Reply:
From testing in the sample app, `maxDuration=0` will let it run forever, and will then be limited by (maxRetries+1)*(delay+jitter), or by the browser-specified timeout for rest calls.

Because this method is only handling setting the text, we can remove the 0 check here. I'll add the appropriate logic in the `__verifyParams` function I'm currently working on.